eslint/babel-eslint-parser/src/parse.cjs

```diff
@@ -13,11 +13,9 @@ let isRunningMinSupportedCoreVersion = null;
 
 module.exports = function parse(code, options, client) {
   // Ensure we're using a version of `@babel/core` that includes `parse()` and `tokTypes`.
-  let minSupportedCoreVersion = ">=7.2.0";
-  // TODO(Babel 8): Update all the version checks
-  if (process.env.BABEL_8_BREAKING) {
-    minSupportedCoreVersion += " || >=8.0.0-0";
-  }
+  const minSupportedCoreVersion = process.env.BABEL_8_BREAKING
+    ? PACKAGE_JSON.version
```

Comment:
```suggestion
    ? ">=8.0.0-0"
```
otherwise we will keep bumping Babel 8 requirements in every babel eslint parser 8 releases.

Reply:
I think during the pretelease period we'd want everything to be exactly at the same version, and then we relax it to ^8.0.0 once stable.